.changeset/dry-apples-suffer.md

```diff
@@ -0,0 +1,6 @@
+---
+'@ai-sdk/google-vertex': patch
```

Comment:
Note though there are no code changes in the `google-vertex` package, this functionality is only user facing in that provider, so I've patch-bumped that module as well for visibility.